lib/transform.js

```diff
@@ -168,3 +169,23 @@ function mapCheckItemIssueToMkdn(issue) {
   }
   return `- [${checked}] #${issue.number}`;
 }
+
+Transform.usernameToId = function (username) {
```

Comment:
I felt pretty good about the idea to make these transform functions, please let me know how its wrong :smiling_face_with_tear:

Reply:
👍